### ISSUE TITLE
Set initial balance for public key

### ISSUE THREAD

**Issue by matthiasgeihs** [ConsenSys-archive/ganache#4468]
As far as I can see, there is currently no way to set an initial balance for a given public key (not secret key).
It would be great to have that option. (Otherwise we need to copy around secret keys which even for testing is not good practice, if it can be avoided.)

Any thoughts?

**Comment by davidmurdoch** [comment]
https://ganache.dev/#evm_setAccountBalance does that do what you want?

**Comment by matthiasgeihs** [comment]
Yes, it does. I forgot to mention that I would ideally want this as a command line argument.

Is there a command line argument corresponding to `setAccountBalance`, so that I can bootstrap a Ganache chain with a given balance for a given public key?
